src/Stratis.BitcoinD/Stratis.BitcoinD.csproj

```diff
@@ -14,6 +14,8 @@
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <NoWarn>1701;1702;1705;IDE0008;</NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsAsErrors>NU1605</WarningsAsErrors>
```

Comment:
I thought you were gonna completely revert this file ?

Reply:
but you haven't reverted it

Reply:
Just done a revert git command line and it's done the same as my change - so not too sure what gains we made here.

   >git checkout 693743e~1 src\Stratis.BitcoinD\Stratis.BitcoinD.csproj

![image](https://user-images.githubusercontent.com/16000683/38895942-4fb84ebc-4289-11e8-9bfb-7785f9e311d0.png)